packages/services-context/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@penumbra-zone/services-context",
```

Comment:
If this kinda depends on https://github.com/penumbra-zone/web/issues/730, think we can hold off publishing until that issue is picked up right?

Reply:
I guess so. Do you think I should revert all changes for `services-context` related to publish?

Reply:
I suppose my comment was that if `services-context` is not really finalized as a name, let's hold off on publishing until that work is picked up. Not sure if anything needs reverted though.